### PR TITLE
feat: Replace message type signatures with CoreMessageType and SendableMessageType

### PR DESCRIPTION
**Pull request by liamcho**
Ticket: https://sendbird.atlassian.net/browse/UIKIT-4358

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
Oh great work! 

Probably it's better to split the PR into 2 to reduce any possible side effects if it's possible. 
1. Replace message type signatures with CoreMessageType and SendableMessageType
ace 
2. Apply the unused import eslint rule

File: src/modules/ChannelList/components/ChannelPreview/index.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ const ChannelPreview: React.FC<ChannelPreviewInterface> = ({
   const isChannelTyping = isTypingIndicatorEnabled && isTyping;
   const isMessageStatusEnabled = isMessageReceiptStatusEnabled
     && (channel?.lastMessage?.messageType === 'user' || channel?.lastMessage?.messageType === 'file')
-    && (channel?.lastMessage as UserMessage | FileMessage)?.sender?.userId === userId;
+    && (channel?.lastMessage as SendableMessageType)?.sender?.userId === userId;
```
Comment (bang9) [inline src/modules/ChannelList/components/ChannelPreview/index.tsx:63]:
```suggestion
    && (channel?.lastMessage as CoreMessageType)?.sender?.userId === userId;
```

File: src/ui/MessageStatus/index.tsx [hunk 2]
```diff
@@ -21,7 +18,7 @@ export const MessageStatusTypes = OutgoingMessageStates;
 
 interface MessageStatusProps {
   className?: string;
-  message: UserMessage | FileMessage;
+  message: SendableMessageType;
```
Comment (bang9) [inline src/ui/MessageStatus/index.tsx:21]: channel.lastMessage 도 받아서 Null 과 CoreMessageType 으로 바뀌어야 할 것 같습니당

```suggestion
  message: CoreMessageType | null;
```

File: src/modules/ChannelList/components/ChannelPreview/index.tsx [hunk 3]
```diff
@@ -142,7 +142,7 @@ const ChannelPreview: React.FC<ChannelPreviewInterface> = ({
                 <MessageStatus
                   className="sendbird-channel-preview__content__upper__last-message-at"
                   channel={channel}
-                  message={channel?.lastMessage as UserMessage | FileMessage}
+                  message={channel?.lastMessage as SendableMessageType}
```
Comment (bang9) [inline src/modules/ChannelList/components/ChannelPreview/index.tsx:145]:
```suggestion
                  message={channel?.lastMessage as CoreMessageType}
```

File: src/modules/App/stories/index.stories.js [hunk 4]
```diff
@@ -533,3 +534,17 @@ export const userListQuery = () => fitPageSize(
...
+    userId={multipleFilesMessageUserId}
+    nickname={multipleFilesMessageUserId}
+    showSearchIcon
+    isMentionEnabled
+    isReactionEnabled
+    isTypingIndicatorEnabledOnChannelList
+    isMessageReceiptStatusEnabledOnChannelList
+    config={{ logLevel: 'all', isREMUnitEnabled: true }}
+  />
+);
```
Comment (HoonBaek) [inline src/modules/App/stories/index.stories.js:550]: What is this for? It seems there are no configs about multiple files message
Could you leave a comment to explain this?

**Review by HoonBaek — APPROVED** [review]
LGTM, thanks for this job

File: src/modules/App/stories/index.stories.js (hunk 4 above)
Comment (liamcho) [reply src/modules/App/stories/index.stories.js:550]: Ah this is not relevant to this PR. Ill remove this.

File: src/ui/MessageStatus/index.tsx (hunk 2 above)
Comment (liamcho) [reply src/ui/MessageStatus/index.tsx:21]: nice catch!

File: src/modules/App/stories/index.stories.js (hunk 4 above)
Comment (liamcho) [reply src/modules/App/stories/index.stories.js]: removed

**Pull request merged**